src/renderer/bootstrap.tsx

```diff
@@ -75,3 +87,4 @@ export async function bootstrap(App: AppComponent) {
 
 // run
 bootstrap(process.isMainFrame ? LensApp : App);
+await attachChromeDebugger();
```

Comment:
This is still broken and will never work `async` doesn't work like this. I also don't understand what you are trying to do? Do you want to block the startup of the application or block the renderering?

Reply:
The aim was to block startup for 1 second, so chrome debugger has time to attach properly. I moved the function to an appropriate place at the top of `bootstrap()`.